recipes/gstreamer/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# gstreamer expects libffi to be in lib64 for some reason.
+mkdir $PREFIX/lib64
+cp -r $PREFIX/lib/libffi* $PREFIX/lib64
```

Comment:
Are you sure this won't result in any hardcoded path later?
Maybe this is easy, like setting a `with-libffi=$PREFIX`, not sure... We need to check the `configure` and `makefile` first.